css/layout.css

```diff
@@ -39,6 +39,15 @@
 #header-residents {
 	background-image: url("../img/header_residents.png");
 }
+#header-residents.header-residents-1 {
+	background-image: url("../img/header_residents1.png");
+}
+#header-residents.header-residents-2 {
+	background-image: url("../img/header_residents2.png");
+}
+#header-residents.header-residents-3 {
+	background-image: url("../img/header_residents3.png");
+}
 #header-data {
```

Comment:
Checked against [https://github.com/jediholo/jedi-static/commit/3d2faf0d63171725c78b71ac9751790dbcd61ca4](this commit) and it looks good to me!